frontend/public/locales/ja/logs.json

```diff
@@ -12,5 +12,6 @@
   "Some lines have been abridged because they are exceptionally long.": "一部の行は非常に長いため、短縮されています。",
   "To view unabridged log content, you can either <2>open the raw file in another window</2> or <6>download it</6>.": "短縮されていないログコンテンツを表示するには、 <2>別のウィンドウで未加工ファイルを開く</2>か、または<6>これをダウンロード</6>します。",
   "The logs for this {{resourceKind}} may be stale.": "この {{resourceKind}} のログは古くなっている可能性があります。",
-  "Refresh": "更新"
-}
\ No newline at end of file
+  "Refresh": "更新",
+  "Wrap lines": "Wrap lines"
```

Comment:
You shouldn't update the ja, ko, or zh locale folder files. `yarn i18n` is all you have to run; it should just update `frontend/public/locales/en/...` for you unless it's a really complicated string. (You may need to fix it if it messes up the pluralization or something like that.)

Reply:
Thanks.  I'll revert the changes and run the yarn command